packages/gasket-nextjs/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# `@gasket/data`
```

Comment:
Are these packages misnamed? I'm showing this is a `@gasket/nextjs` package, not `@gasket/data`

Reply:
Good catch, I fixed it.